ext/cron/01_cron.ts

```diff
@@ -3,9 +3,73 @@
 // @ts-ignore internal api
 const core = Deno.core;
 
+function formateToCronSchedule(
```

Comment:
Would it be possible to add unit tests specifically for this function? This way you can test many different cron expressions without having to run end-to-end `Deno.cron` tests.

Reply:
Good idea, added test cases of `formateToCronSchedule`